### ISSUE TITLE
Unable to Drag the Tab outside of the window to spawn a new window using keyboard.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#581]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Control Gallery app.
2. Navigate to TabView button and activate it.
3. Navigate to "Click here to launch the sample" button->Activate it->Turn on ProgressRing toggle switch->Try to Drag the tab outside of the window to spawn a new window and Observe. 

**Expected behavior**

User should be able to create new window by dragging a tab. (or) there should be option to user to do this functionality like open a new window by using context menu on the tab etc.

**Actual behavior**

User is unable to create new window.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97395653-a19b2500-18a2-11eb-80f9-5f1c1477fa09.png)

**Version Info**

Gallery version (found on Settings page): 1.2.10.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/23669905)

**Comment by YuliKl** [comment]
Comment in the original issue:
Probably the sample needs a context menu for "move to new window" on the tab, to match Edge behavior? Maybe a spec issue if we think this is something the control should do automatically. (@stmoy FYI)

**Comment by stmoy** [comment]
We've avoided baking a context menu into the control, so I think the proper way to solve this at this time would be for the app to add a context menu. This is generally the guidance anyway, so I'd say the sample is incomplete. (whoops 😅)

**Comment by koeries** [comment]
Fuck u pussy
U fucking stupid1
